include/mpsl/mpsl_assert.rst

```diff
@@ -0,0 +1,25 @@
+.. _mpsl_assert:
+
+Multiprotocol Service Layer assert
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Multiprotocol Service Layer assert library makes it possible to add a custom assert handler to the :ref:nrfxlib:mpsl library.
+You can then use this assert handler to print custom error messages or log assert information.
+
+:option:`CONFIG_MPSL_ASSERT_HANDLER` enables the custom assert handler.
+If enabled, the application must provide the definition of :c:func:`mpsl_assert_handle`.
+The :c:func:`mpsl_assert_handle` function is invoked whenever the MPSL code encounters an unrecoverable error.
+
+API documentation
+*****************
+
+.. doxygengroup:: mpsl_assert
+   :project: nrf
+   :members:
+
+
+| Header file: :file:`include/mpsl/mpsl_assert.h`
```

Comment:
This must be moved about the doxygengroup.

Reply:
updated